src/go/rpk/pkg/cli/cmd/redpanda/start.go

```diff
@@ -1138,3 +1168,41 @@ func setClusterProperties(fs afero.Fs, cfgFileLocation string, properties map[st
 
 	return nil
 }
+
+func parseSandboxFlag(args []string, sandbox *string) {
+	for i := 0; i < len(args)-1; i++ {
+		if args[i] == "--sandbox" && !strings.HasPrefix(args[i+1], "-") {
+			*sandbox = args[i+1]
+		}
+	}
+}
+
+const helpSandbox = `Sandbox mode sets relevant properties for local development or 
```

Comment:
I think this is wrapping with the `const helpSandbox = ` prefix; checking locally, `test container` moves to the prior line if I wrap at 80

Also, do we have any online docs we can refer to here? cc. @Feediver1

Reply:
You are right, wrapping at 80 based on text only:

```
const helpSandbox = `Sandbox mode sets relevant properties for local development or test container
usage.
```